tap4j/src/main/java/org/tap4j/parser/Tap13Parser.java

```diff
@@ -32,22 +32,11 @@
 import java.nio.charset.Charset;
 import java.nio.charset.CharsetDecoder;
 import java.nio.charset.UnsupportedCharsetException;
-import java.util.Map;
-import java.util.Scanner;
-import java.util.Stack;
+import java.util.*;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import org.tap4j.model.BailOut;
-import org.tap4j.model.Comment;
-import org.tap4j.model.Footer;
-import org.tap4j.model.Header;
-import org.tap4j.model.Plan;
-import org.tap4j.model.TapElement;
-import org.tap4j.model.TapElementFactory;
-import org.tap4j.model.TestResult;
-import org.tap4j.model.TestSet;
-import org.tap4j.model.Text;
+import org.tap4j.model.*;
```

Comment:
I can re-enter the imports later, but if you could change to specific imports that'd be simpler. I know it sounds nit-picking, but at work we actually asked another dev that works on an IDE that was doing that. While doing a refactoring of old code, even her IDE would fail to identify the classes that it had to use, as we had a few duplications :cry:

Reply:
Sure, I'll update it soon.